src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/deconvolution/deconvolution_kernel_b_fs_zyx_fsv16.cpp

```diff
@@ -283,4 +275,51 @@ JitConstants DeconvolutionKernel_b_fs_zyx_fsv16::GetJitConstants(const deconvolu
     return jit;
 }
 
+KernelsData DeconvolutionKernel_b_fs_zyx_fsv16::GetKernelsData(const Params& params, const optional_params& options) const {
+    assert(params.GetType() == KernelType::DECONVOLUTION);
+
+    if (!Validate(params, options)) {
+        return{};
+    }
+
+    const deconvolution_params& orgParams = static_cast<const deconvolution_params&>(params);
+    DispatchData dispatchData = SetDefault(orgParams);
+    KernelData kd = KernelData::Default<deconvolution_params>(params);
+    deconvolution_params& newParams = *static_cast<deconvolution_params*>(kd.params.get());
+
+    bool succeed = UpdateWeightsParams(newParams,
+                                       options,
+                                       GetPreferredWeightsLayout(newParams),
+                                       kd.weightsReorderParams,
+                                       GetSupportedKey(),
+                                       newParams.groups);
+
+    if (!succeed) {
+        return {};
```

Comment:
if most of this checks are from the parent class, then you can just do Parent::Validate(params, options) call for them.